pkg/goDB/DBWorkManager.go

```diff
@@ -473,8 +570,15 @@ func (w *DBWorkManager) readMetadataAndEvaluate(workDir *gpfile.GPDir, blocks []
 	return aggMetadata, nil
 }
 
+func logWorkloadStats(logger *logging.L, msg string, stats *WorkloadStats) {
+	if stats == nil {
+		return
+	}
+	logger.With("stats", stats).Info(msg)
+}
+
 // main query processing
-func (w *DBWorkManager) grabAndProcessWorkload(ctx context.Context, wg *sync.WaitGroup, workloadChan <-chan DBWorkload, mapChan chan hashmap.AggFlowMapWithMetadata) {
+func (w *DBWorkManager) grabAndProcessWorkload(ctx context.Context, id int, wg *sync.WaitGroup, workloadChan <-chan DBWorkload, mapChan chan hashmap.AggFlowMapWithMetadata) {
```

Comment:
What does the `id` do, is it even used (hard to find here, `id` isn't that specific a string :see_no_evil: )?

Reply:
It's used to identify which processing unit is currently grabbing and processing workloads. It's used in the log messages.

Will make clearer in feedback.